dbtwiz/admin/cleanup.py

```diff
@@ -0,0 +1,112 @@
+from dbtwiz.auth import ensure_auth
+from dbtwiz.manifest import Manifest
+from dbtwiz.target import Target
+from dbtwiz.logging import info, error
+
+from pathlib import Path
+from google.cloud import bigquery
+
+
+def cleanup_development_dataset(force_delete: bool) -> None:
+    """Delete all materializations in the development dataset"""
+    ensure_auth()
+
+    Manifest.update_manifests("dev")
+    manifest = Manifest()
+
+    # Get project and dataset from first materialization in dev manifest
+    project, dataset = [
+        (m["database"], m["schema"])
+        for m in manifest.models().values()
+        if m["materialized"] != "ephemeral"
+    ][0]
+    bq_client = bigquery.Client(project=project)
+    tables = list(bq_client.list_tables(dataset))
+    if not tables:
+        info(f"Dataset {project}.{dataset} is already empty.")
+        return
+    info(f"There are {len(list(tables))} tables in the {project}.{dataset} dataset.")
+    if not force_delete:
+        answer = input("Delete all tables? (y/N)? ")
+        if not answer.lower() in ["y", "yes"]:
+            return
+    for table in tables:
+        try:
+            bq_client.delete_table(table)
```

Comment:
Does the bq_client distinguish between views and tables? If not then it might be informative to state what materialization that is being delete, like "Deleted view/table"

Reply:
It doesn't, and this method works for both. But we can distinguish between them when logging what we do.